### DISCUSSION TITLE
Installation with Fermitools

### DISCUSSION THREAD

**Discussion by PepeJoseHU** [threeML/threeML#576] [General]
Hello, I plan to install 3ML for Fermi data with CONDA, but looks 3ML have dependence with fermitools. But when I install 'fermitools', it also make an environment with CONDA, so how to fix this problem or if there have way to make connection between different CONDA environments?

Thanks!

**Comment by grburgess** [comment]
This is not how virtual environments work. You must install all the software you want to use inside the same virtual environment

**Comment by grburgess** [comment]
This is an installation issue and should be moved to the issue tracker.
